modules/customize-certificates-add-service-serving-apiservice.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-apiservice_{context}"]
+= Add a service certificate to an APIService
+
+////
+
+// TODO: intro + important note?
+
+A Pod can access the service CA certificate by mounting a ConfigMap that
```

Comment:
@marun For each of these new resources, I've commented out the intro paragraph, an [IMPORTANT] block and a [NOTE] block (each still relevant for ConfigMaps). Can you help me with appropriate introductions for each, and if either of these blocks are still applicable for any of the resources?

Reply:
As per our conversation, 'CA certificate' -> 'service CA bundle' if not talking about service/secret.